.circleci/config.yml

```diff
@@ -37,4 +37,4 @@ jobs:
           destination: artifacts
       - store_test_results:
           path: ~/junit/
-          destination: testartifacts
\ No newline at end of file
+          destination: testartifacts
```

Comment:
I think we should revert the change in this file

Reply:
good catch, thank you